internal/controller/routerconfiguration/underlay_vni_controller.go

```diff
@@ -81,6 +81,10 @@ func (r *PERouterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		slog.Error("failed to fetch router pod", "node", r.MyNode, "error", err)
 		return ctrl.Result{}, err
 	}
+	multusEnabled := false
+	if _, ok := routerPod.Annotations["k8s.v1.cni.cncf.io/networks"]; ok {
```

Comment:
what if we make this a parameter? We'll have to drive the annotation from the chart anyways.